### PR TITLE
fix: wrong cli description

### PR DESCRIPTION
**Pull request by yyzxw**
**What this PR does / why we need it**:

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

File: cmd/helm/plugin_install.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func newPluginInstallCmd(out io.Writer) *cobra.Command {
 	o := &pluginInstallOptions{}
 	cmd := &cobra.Command{
 		Use:     "install [options] <path|url>...",
-		Short:   "install one or more Helm plugins",
+		Short:   "install one Helm plugins",
```
Comment (sabre1041) [inline cmd/helm/plugin_install.go:43]:
```suggestion
		Short:   "install a Helm plugin",
```

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Good catch. Proposed a grammatical change

File: cmd/helm/plugin_install.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ func newPluginInstallCmd(out io.Writer) *cobra.Command {
 	o := &pluginInstallOptions{}
 	cmd := &cobra.Command{
 		Use:     "install [options] <path|url>...",
```
Comment (joejulian) [inline cmd/helm/plugin_install.go:42]: If this is true, then this should remove the ellipsis.

**Review by joejulian — CHANGES_REQUESTED** [review]

File: cmd/helm/plugin_install.go (hunk 2 above)
Comment (yyzxw) [reply cmd/helm/plugin_install.go]: done

**Review by mattfarina — APPROVED** [review]

**Comment by mattfarina** [comment]
Merging as the requested changes have been addressed. Thanks for the fix.

**Comment by mattfarina** [comment]
Note, merging with one maintainer approval since this is XS and that's allowed at this size of a change.

**Pull request merged**